openshift/operator-controller/manifests/14-rolebinding-openshift-config-operator-controller-manager-rolebinding.yml

```diff
@@ -13,5 +13,5 @@ roleRef:
   name: operator-controller-manager-role
 subjects:
   - kind: ServiceAccount
-    name: operator-controller-controller-manager
+    name: controller-manager
```

Comment:
We should not change the name, right?

Reply:
must've been tangled during a rebase, good spot